File: ui/article-aem-renderer/src/test/java/org/cru/godtools/article/aem/service/AEMDownloadManagerTest.java
package org.cru.godtools.article.aem.service;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class AEMDownloadManagerTest {
    @Test
    public void testRoundTimestamp() {
        assertEquals(15000, AEMDownloadManger.roundTimestamp(15234, 1000));
    }
}

File: ui/article-aem-renderer/src/test/java/org/cru/godtools/article/aem/service/AEMDownloadManagerTest.kt
package org.cru.godtools.article.aem.service

import org.junit.Assert.assertEquals
import org.junit.Test

class AEMDownloadManagerTest {
    @Test
    fun testRoundTimestamp() {
        assertEquals(15000, AemArticleManger.roundTimestamp(15234, 1000))
    }
}
